### ISSUE TITLE
Изменения в первой программе

### ISSUE THREAD

**Issue by justjune** [TrubnikovGI/AiSD2020#1]
- [x] 1) Убрать приглашения для интерпретатора

- [x] 2) Исправить синтаксические ошибки

- [x] 3) Вывод в файле программы не нужен

- [ ] 4) Программа должна иметь расширение py

**Comment by TrubnikovGI** [comment]
я не понимаю что такое приглашение для интерпретатора, и разве файл имеет другое разрешение отличное от py

**Comment by justjune** [comment]
![image](https://user-images.githubusercontent.com/41794497/102205668-50e09980-3eed-11eb-88d2-49db219f16a7.png)
Строчки выше --- текст операционной системы. Это не код на python.

**Comment by justjune** [comment]
">>> " --- это не команда Python, а приглашение, которое печатает интерпретатор, чтобы вы ввели очередную команду. В файле с программой этого быть не должно. Какой средой для исполнения файлов пользуетесь?

**Comment by TrubnikovGI** [comment]
Так, понятнее стало. Насколько я могу судить из того что мне посоветовали сделать одногруппники среда разроботки называется IDLE

**Comment by justjune** [comment]
Эта среда, если в ней открыть файл с расширением py,  работает так же, как Visual Studio, только меню другие. Кроме того, можно выполнять работы без установки python на (https://repl.it)

**Comment by TrubnikovGI** [comment]
Понял, сейчас так и делаю. Постараюсь в скором времени написать программу с записью температур в файл и отправлю вам ссылку.

**Comment by TrubnikovGI** [comment]
На всякий случай дублирую сюда ссылку со страницы OneNote с записью данных в файл https://repl.it/@TrubnikovG/tiempieratura-riek#main.py
